<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>884</width>
    <height>678</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="liste">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>1251</width>
      <height>491</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QLabel{
color:white;
font-size:16px;
}
QPushButton
{
background:red;
border-radius:16px;
}
QPushButton:hover {
color:blue;
border-size:20px;
background:#333
}
QPushButton{
color:white;
border-radius:15px;
}
QLineEdit
{
background:transparent;
border:none;
color:#b0b0b0;
font-size:16px;
border-bottom:1px solid #fefff4;
}





</string>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <property name="tabBarAutoHide">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Liste de mat√©riel</string>
     </attribute>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>10</y>
        <width>871</width>
        <height>521</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame{
background:white;

}
QLabel{
color:white;
font-size:16px;
}
QPushButton
{
background:red;
border-radius:16px;
}
QPushButton:hover {
transform: scale(5); 
width : 250px;
color:red;
}
QPushButton{
color:white;
border-radius:15px;
}
QLineEdit
{
background:transparent;
border:none;
color:#a0a0a0;
font-size:16px;
border-bottom:1px solid #333;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="ButtonSupprimer">
       <property name="geometry">
        <rect>
         <x>510</x>
         <y>60</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:#001F26;
</string>
       </property>
       <property name="text">
        <string>supprimer</string>
       </property>
      </widget>
      <widget class="QPushButton" name="ButtonFiltre">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>60</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:#001F26;</string>
       </property>
       <property name="text">
        <string>Filtre par location</string>
       </property>
      </widget>
      <widget class="QPushButton" name="ButtonTri">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>60</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:#001F26;</string>
       </property>
       <property name="text">
        <string>Tri par nom</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="recherche">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>181</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border:2px solid color rgb(170, 85, 127);
border-radius:10px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Recherche...</string>
       </property>
      </widget>
      <widget class="QTableView" name="tableView">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>100</y>
         <width>451</width>
         <height>211</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit">
       <property name="geometry">
        <rect>
         <x>610</x>
         <y>60</y>
         <width>161</width>
         <height>24</height>
        </rect>
       </property>
       <property name="placeholderText">
        <string>reference a supprimer</string>
       </property>
      </widget>
      <widget class="QPushButton" name="ButtonAjouter_4">
       <property name="geometry">
        <rect>
         <x>630</x>
         <y>380</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:#001F26;</string>
       </property>
       <property name="text">
        <string>modifler</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="ReferenceLE_3">
       <property name="geometry">
        <rect>
         <x>630</x>
         <y>110</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="placeholderText">
        <string>R√©ference de mat√©riel</string>
       </property>
      </widget>
      <widget class="QLabel" name="EtatAjout_3">
       <property name="geometry">
        <rect>
         <x>540</x>
         <y>280</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color:black;</string>
       </property>
       <property name="text">
        <string>Etat location:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="ReferenceLE_5">
       <property name="geometry">
        <rect>
         <x>640</x>
         <y>320</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
       <property name="placeholderText">
        <string>saisir le prix en DT</string>
       </property>
      </widget>
      <widget class="QLabel" name="Refrence_6">
       <property name="geometry">
        <rect>
         <x>540</x>
         <y>230</y>
         <width>81</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color:black;</string>
       </property>
       <property name="text">
        <string>Quantite</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="MatriculeLE_2">
       <property name="geometry">
        <rect>
         <x>630</x>
         <y>180</y>
         <width>211</width>
         <height>31</height>
        </rect>
       </property>
       <property name="placeholderText">
        <string>saisir le matricule</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="ReferenceLE_6">
       <property name="geometry">
        <rect>
         <x>620</x>
         <y>220</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="placeholderText">
        <string>qtt</string>
       </property>
      </widget>
      <widget class="QLabel" name="Refrence_3">
       <property name="geometry">
        <rect>
         <x>540</x>
         <y>120</y>
         <width>81</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color:black;</string>
       </property>
       <property name="text">
        <string>R√©f√©rence:</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_3">
       <property name="geometry">
        <rect>
         <x>640</x>
         <y>280</y>
         <width>75</width>
         <height>24</height>
        </rect>
       </property>
       <property name="currentText">
        <string>loue</string>
       </property>
       <item>
        <property name="text">
         <string>loue</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>non loue</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="Matricule_2">
       <property name="geometry">
        <rect>
         <x>540</x>
         <y>200</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color:black;</string>
       </property>
       <property name="text">
        <string>Matricule:</string>
       </property>
      </widget>
      <widget class="QLabel" name="NomAjout_2">
       <property name="geometry">
        <rect>
         <x>540</x>
         <y>160</y>
         <width>46</width>
         <height>13</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color:black;</string>
       </property>
       <property name="text">
        <string>Nom:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="nomLE_2">
       <property name="geometry">
        <rect>
         <x>630</x>
         <y>140</y>
         <width>211</width>
         <height>31</height>
        </rect>
       </property>
       <property name="placeholderText">
        <string>saisir le nom du matriel</string>
       </property>
      </widget>
      <widget class="QLabel" name="Refrence_7">
       <property name="geometry">
        <rect>
         <x>540</x>
         <y>330</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color:black;</string>
       </property>
       <property name="text">
        <string>prix(DT/jour):</string>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="geometry">
        <rect>
         <x>510</x>
         <y>100</y>
         <width>341</width>
         <height>321</height>
        </rect>
       </property>
       <property name="title">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="Buttonrecherche">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>62</y>
         <width>21</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:white;</string>
       </property>
       <property name="text">
        <string>üîç</string>
       </property>
      </widget>
      <widget class="QPushButton" name="PDF">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>350</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:#001F26;</string>
       </property>
       <property name="text">
        <string>PDF</string>
       </property>
      </widget>
      <widget class="QPushButton" name="logoAjout_3">
       <property name="geometry">
        <rect>
         <x>710</x>
         <y>-40</y>
         <width>221</width>
         <height>181</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
background:transparent;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../../c_logo.png</normaloff>../../../c_logo.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_4">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>20</y>
         <width>75</width>
         <height>24</height>
        </rect>
       </property>
       <property name="currentText">
        <string>loue</string>
       </property>
       <item>
        <property name="text">
         <string>loue</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>non loue</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="filtre">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>20</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color:black;</string>
       </property>
       <property name="text">
        <string>Filtrer par</string>
       </property>
      </widget>
      <zorder>groupBox_3</zorder>
      <zorder>ButtonSupprimer</zorder>
      <zorder>ButtonFiltre</zorder>
      <zorder>ButtonTri</zorder>
      <zorder>recherche</zorder>
      <zorder>tableView</zorder>
      <zorder>lineEdit</zorder>
      <zorder>ButtonAjouter_4</zorder>
      <zorder>ReferenceLE_3</zorder>
      <zorder>EtatAjout_3</zorder>
      <zorder>ReferenceLE_5</zorder>
      <zorder>Refrence_6</zorder>
      <zorder>MatriculeLE_2</zorder>
      <zorder>ReferenceLE_6</zorder>
      <zorder>Refrence_3</zorder>
      <zorder>comboBox_3</zorder>
      <zorder>Matricule_2</zorder>
      <zorder>NomAjout_2</zorder>
      <zorder>nomLE_2</zorder>
      <zorder>Refrence_7</zorder>
      <zorder>Buttonrecherche</zorder>
      <zorder>PDF</zorder>
      <zorder>logoAjout_3</zorder>
      <zorder>comboBox_4</zorder>
      <zorder>filtre</zorder>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <property name="styleSheet">
      <string notr="true">QFrame{
background:white;

}
QLabel{
color:white;
font-size:16px;
}
QPushButton
{
background:red;
border-radius:16px;
}
QPushButton:hover {
transform: scale(5); 
width : 250px;
color:red;
}
QPushButton{
color:white;
border-radius:15px;
}
QLineEdit
{
background:transparent;
border:none;
color:#a0a0a0;
font-size:16px;
border-bottom:1px solid #333;
}</string>
     </property>
     <attribute name="title">
      <string>Mat√©riel</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>801</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>13</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame{
background:white;

}
QLabel{
color:white;
font-size:16px;
}
QPushButton
{
background:red;
border-radius:16px;
}
QPushButton:hover {
transform: scale(5); 
width : 250px;
color:red;
}
QPushButton{
color:white;
border-radius:15px;
}
QLineEdit
{
background:transparent;
border:none;
color:#a0a0a0;
font-size:16px;
border-bottom:1px solid #333;
}</string>
      </property>
      <property name="title">
       <string>Ajouter mat√©riel</string>
      </property>
     </widget>
     <widget class="QFrame" name="frame_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>831</width>
        <height>491</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame{
background:white;

}
QLabel{
color:black;
font-size:16px;
}
QPushButton
{
background:red;
border-radius:16px;
}
QPushButton:hover {
transform: scale(5); 
width : 250px;
color:red;
}
QPushButton{
color:white;
border-radius:15px;
}
QLineEdit
{
background:transparent;
border:none;
color:#a0a0a0;
font-size:16px;
border-bottom:1px solid #333;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="ButtonAjouter_2">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>390</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:#001F26;</string>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
      <widget class="QLabel" name="Matricule">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>140</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Matricule:</string>
       </property>
      </widget>
      <widget class="QLabel" name="NomAjout">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>100</y>
         <width>46</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>Nom:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="nomLE">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>80</y>
         <width>211</width>
         <height>31</height>
        </rect>
       </property>
       <property name="placeholderText">
        <string>saisir le nom du matriel</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="MatriculeLE">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>120</y>
         <width>211</width>
         <height>31</height>
        </rect>
       </property>
       <property name="placeholderText">
        <string>saisir le matricule</string>
       </property>
      </widget>
      <widget class="QLabel" name="EtatAjout_2">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>220</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Etat location:</string>
       </property>
      </widget>
      <widget class="QLabel" name="Refrence_2">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>270</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>prix(DT/jour):</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="ReferenceLE_2">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>260</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
       <property name="placeholderText">
        <string>saisir le prix en DT</string>
       </property>
      </widget>
      <widget class="QLabel" name="Refrence_4">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>320</y>
         <width>141</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Ajouter une image:</string>
       </property>
      </widget>
      <widget class="QPushButton" name="ButtonAjouter_3">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>320</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:#cece00;
border-radius:15px;</string>
       </property>
       <property name="text">
        <string>Upload IMG</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_2">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>220</y>
         <width>75</width>
         <height>24</height>
        </rect>
       </property>
       <property name="currentText">
        <string>loue</string>
       </property>
       <item>
        <property name="text">
         <string>loue</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>non loue</string>
        </property>
       </item>
      </widget>
      <widget class="QLineEdit" name="ReferenceLE_4">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>160</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="placeholderText">
        <string>qtt</string>
       </property>
      </widget>
      <widget class="QLabel" name="Refrence_5">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>170</y>
         <width>81</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Quantite</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="ReferenceLE">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>50</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>R√©ference de mat√©riel</string>
       </property>
      </widget>
      <widget class="QLabel" name="Refrence">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>60</y>
         <width>81</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>R√©f√©rence:</string>
       </property>
      </widget>
      <widget class="QPushButton" name="logoAjout_4">
       <property name="geometry">
        <rect>
         <x>680</x>
         <y>-50</y>
         <width>221</width>
         <height>181</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
background:transparent;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../../c_logo.png</normaloff>../../../c_logo.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <zorder>frame_2</zorder>
     <zorder>groupBox_2</zorder>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Statistiques</string>
     </attribute>
     <widget class="QFrame" name="frame_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>831</width>
        <height>491</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame{
background:white;

}
QLabel{
color:black;
font-size:16px;
}
QPushButton
{
background:red;
border-radius:16px;
}
QPushButton:hover {
transform: scale(5); 
width : 250px;
color:red;
}
QPushButton{
color:white;
border-radius:15px;
}
QLineEdit
{
background:transparent;
border:none;
color:#a0a0a0;
font-size:16px;
border-bottom:1px solid #333;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="logoAjout_5">
       <property name="geometry">
        <rect>
         <x>680</x>
         <y>-50</y>
         <width>221</width>
         <height>181</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
background:transparent;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../../c_logo.png</normaloff>../../../c_logo.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>70</y>
         <width>501</width>
         <height>341</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout"/>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>884</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
